keiko-redis-spring/src/main/kotlin/com/netflix/spinnaker/config/RedisQueueConfiguration.kt

```diff
@@ -46,6 +46,10 @@ import java.util.Optional
 
 @Configuration
 @EnableConfigurationProperties(RedisQueueProperties::class)
+@ConditionalOnProperty(
+  value = ["keiko.queue.sql.enabled"],
+  havingValue = "false",
+  matchIfMissing = true)
```

Comment:
We probably shouldn't do it this way. Having one backend reference another is sketch town. I'd rather see a setting explicit for Redis, and one explicit for SQL, where the Redis backend is the default (for the sake of backwards compat).